File: day25.java
import java.io.*;
import java.util.*;
import java.text.*;
import java.math.*;
import java.util.regex.*;

public class Solution {

    public static void main(String[] args) {
        /* Enter your code here. */
        Scanner sc=new Scanner(System.in);
        int n=sc.nextInt();
        for (int i=0; i<n ;i++){
            int number = sc.nextInt();
            if(checkPrime(number)){
                System.out.println("Prime");
            }else{
                System.out.println("Not prime");
            }
        }
    }

    public static boolean checkPrime(int number){
        if (number <=3){
            return number > 1;
        }
        if((number % 2 ==0) || (number % 3 == 0) ){
            return false;
        }
        int i=5;
        while (Math.pow(i, 2) <= number){
            if((number % i == 0) ||(number % (i+2) ==0)){
                return false;
            }
           i+=6;
        }
    return true;
    }
}

File: day25.py
# Enter your code here. Read input from STDIN. Print output to STDOUT
import math
def is_prime(n):
    if n <= 3:
        return n > 1
    if n % 2 == 0 or n % 3 == 0:
        return False
    i = 5
    while i ** 2 <= n:
        if n % i == 0 or n % (i + 2) == 0:
            return False
        i += 6
    return True

T=int(input())
for i in range(T):
    number = int(input())
    if(is_prime(number)):
        print("Prime")
    else:
        print("Not prime")
